.github/workflows/ci-actions.yml

```diff
@@ -76,15 +76,15 @@ jobs:
           KEYCLOAK_PASSWORD: admin
           JAVA_OPTS: "-server -Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true -Djava.awt.headless=true -Dkeycloak.profile.feature.upload_scripts=enabled"
         ports:
-          - 8180:8080
+          - 127.0.0.1:8180:8080
```

Comment:
Out of curiosity what's the story behind this one?

Reply:
We had some weird failures with SQL Server not being accessible while it was properly started, I'd like to see if it improves things or not.

Reply:
It's worth a try I guess, but I'm curious to why we thing that this will solve the issue :)

Reply:
I know Sanne tried that at some point so that's why I thought of it.

Reply:
OK Cool. Lets hope it works